testing/scripts/test_helpdesk_pw.py

```
@@ -2589,34 +2606,69 @@ def test_47_inbound_email_webhook(page: Page):
+        # Apache strips the Authorization header from external requests in some
+        # configurations; going through Docker's internal network avoids this.
+        repo_root = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", ".."))
+        php_code = (
+            "$r=wp_remote_post('http://wordpress/?rest_route=/swh/v1/inbound-email',"
+            "['sslverify'=>false,'timeout'=>15,'redirection'=>0,"
+            f"'headers'=>['Authorization'=>'Bearer {test_secret}'],"
+            f"'body'=>['subject'=>'{subject}','from'=>'{ticket_email}','body-plain'=>'{body}']]);"
+            "if(is_wp_error($r)){echo 'WP_ERROR:'.$r->get_error_message();}"
+            "else{echo wp_remote_retrieve_response_code($r).\"\\n\".wp_remote_retrieve_body($r);}"
+        )
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)